statsd/src/main/java/com/avast/metrics/statsd/StatsDCounter.java

```diff
@@ -30,31 +30,41 @@ public String getName() {
     @Override
     public void inc() {
         count.incrementAndGet();
-        client.count(name, 1, sampleRate);
+        underlying(1);
     }
 
     @Override
     public void inc(final long n) {
         count.addAndGet(n);
-        client.count(name, n, sampleRate);
+        underlying(n);
     }
 
     @Override
     public void dec() {
         count.decrementAndGet();
-        client.count(name, -1, sampleRate);
+        underlying(-1);
     }
 
     @Override
     public void dec(final int n) {
         final int delta = -n;
 
         count.addAndGet(delta);
-        client.count(name, delta, sampleRate);
+        underlying(delta);
     }
 
     @Override
     public long count() {
         return count.get();
     }
+
+
+    @Override
+    public void init() {
+        underlying(0);
```

Comment:
Maybe you misinterpreted my comment. Please ensure `sampleRate` is not used (`1.0` is used instead) just for `init` method. Other usages of the `sampleRate` are OK :)